Detectors/MUON/MCH/Simulation/src/Digitizer.cxx

```diff
@@ -57,7 +57,7 @@ const o2::mch::mapping::Segmentation& segmentation(int detElemId)
 
 bool isStation1(int detID)
 {
-  return detID < 400;
+  return detID < 300;
```

Comment:
Is this the fix you mentioned in the commit text?

Reply:
Yes indeed.